### ISSUE TITLE
Left and right side of imported PDF slides are cut off in the Control window

### ISSUE THREAD

**Issue by maehne** [OpenBoard-org/OpenBoard#846]
In OpenBoard 1.7.0 (built using the AUR package on Arch Linux x86_64 and forced to use XWayland for rendering), the left and right of imported PDF slides are cut off in the Control window, when it is on a ultrawide screen (3440 x 1440 pixel). The slides are correctly diplayed in the Display window (1920 x 1200 pixel):

![grafik](https://github.com/OpenBoard-org/OpenBoard/assets/1174243/cf96a8e8-2651-48ad-8ed7-548a68be5b2c)

The PDF slides have been prepared using the LaTeX Beamer document class. OpenBoard sets in this case the page size always to "Custom size". If I change it to 16:9, only the current slide becomes smaller and covers only about 1/3 of the screen:

![grafik](https://github.com/OpenBoard-org/OpenBoard/assets/1174243/9941a13a-99fb-4073-8231-323ba368a4e7)

Personally, I would prefer that the whole slide is alway visible in the Control window - even if it does not fit into the Display window. A transparent frame or border line suffices in my eyes to indicate the visible screen area.

**Comment by letsfindaway** [comment]
I think this is the same as #887. See also my PR #901.

But I think your idea of a transparent frame is appealing: Just give an indication what is visible on the display screen instead of the boxing as it is currently done. Will think about this and how it should behave. Should full interaction be possible also on the outer area?

**Comment by maehne** [comment]
I agree that issue #887 is a duplicate of this one. IMHO, being able to interact with the slides (writing, erasing, etc.) with the slides outside the visible area is desirable, e.g, when you are approaching the boarder while writing. One would like to finish the annotation while speaking and then move that part into the visible area. Otherwise, the annotation would get cut off and it looks ugly if you then try to complete it in second attempt.

Besides a grayed-out area indicating the non-visible part, maybe some people would prefer a simple (dashed) rectangle as a (configurable) option.
